### PR TITLE
MB-8334 Counselors shouldn't be able to update shipment status

### PR DESCRIPTION
**Pull request by mlm55**
## Description

The `UpdateMTOShipment` handler is meant for counselors to update shipments. However, counselors should not be able to make any status changes to shipments. This pr removes status changes from the handler.

## Setup

```sh
make server_run
```
Set up [postman](https://github.com/transcom/mymove/wiki/Test-Admin-and-Office-APIs-with-Postman) and hit `http://officelocal:3000/ghc/v1/move_task_orders/e0463784-d5ea-4974-b526-f2a58c79ed07/mto_shipments/3c207b2a-d946-11eb-b8bc-0242ac130003` with a body containing:
```
{
    "counselorRemarks": "New customer remarks",
    "status": "APPROVED"
}
```

Counselor remarks should be updated, while status should be ignored

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8334) for this change

### CODE REVIEW

File: pkg/handlers/ghcapi/internal/payloads/payload_to_model.go [hunk 1]
```diff
@@ -166,7 +166,6 @@ func MTOShipmentModelFromUpdate(mtoShipment *ghcmessages.UpdateShipment) *models
 		RequestedDeliveryDate: &requestedDeliveryDate,
 		CustomerRemarks:       mtoShipment.CustomerRemarks,
 		CounselorRemarks:      mtoShipment.CounselorRemarks,
-		Status:                models.MTOShipmentStatus(mtoShipment.Status),
```
Comment (YanZ777) [inline pkg/handlers/ghcapi/internal/payloads/payload_to_model.go:169]: Did you happen to verify that the status wasn't being sent for things that do update the shipment? e.g. Service counseling forms?

Comment (mlm55) [reply pkg/handlers/ghcapi/internal/payloads/payload_to_model.go:169]: i think `ServicesCounselingEditShipmentDetails.jsx` is the only page using this endpoint and it doesn't look like it sends a status. `ServicesCounselingShipmentForm.jsx` uses a different endpoint to update statuses.

Comment (felipe-lee) [reply pkg/handlers/ghcapi/internal/payloads/payload_to_model.go:169]: Looks like `ServicesCounselingShipmentForm.jsx` just gets passed a handler by `ServicesCounselingEditShipmentDetails.jsx` and it passes this as the body
```js
const formattedMtoShipment = {
  moveTaskOrderID: moveId,
  shipmentType,
  customerRemarks,
  counselorRemarks,
  agents: [],
};
```
Plus optionally setting these: 
```js
requestedPickupDate
pickupAddress
requestedDeliveryDate
destinationAddress
secondaryPickupAddress
secondaryDeliveryAddress
```
So i think we're good on that end.

**Review by felipe-lee — APPROVED** [review]
LGTM. I tried sending a patch request to the endpoint with an update to the status and it just ignored it, as expected.

**Pull request merged**